scripts/scripts.js

```diff
@@ -310,6 +310,26 @@ export function getWindowSize() {
     height: windowHeight,
   };
 }
+
+export function addTopSpacingStyleToFirstMatchingSection(main) {
+  const excludedClasses = ['static', 'spacer-container', 'feed-container', 'modal-fragment-container', 'hero-banner-container', 'hero-career-container', 'breadcrumb-container', 'hero-horizontal-tabs-container', 'carousel-container'];
+  const sections = [...main.querySelectorAll(':scope > div')];
+  let added = false;
+
+  sections.every((section) => {
+    if (added || sections.indexOf(section) >= 3) return false;
```

Comment:
What's special about "3" over here ?

Reply:
One of the first 3 sections is a candidate for this;
The first is probably gonna be the `breadcrumb-container`, the second one can be a `hero`, and the third is the last possible candidate
https://global-top-spacing--sunstar--hlxsites.hlx.page/about/business-performance <-here's an example
If we reach the 4th section we just stop trying to find one basically
It should be `=== 3 `; and I'll also move this to a variable

Reply:
Ok maybe add an explanatory comment along this lines otherwise it would be tough for someone to understand without context.

Reply:
Yep, I'll add a comment